app/js/displayHideGraph.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * This function toggles .hidden on the graph and table.
+ * It also toggles the class on the buttons to display the button with the correct text on it.
+ *
+ */
+
+function toggleGraphDisplay() {
+    let graphOnPage = document.querySelector('.show-graph');
+    graphOnPage.classList.toggle('hidden');
+    let tableOnPage = document.querySelector('.user-score-table');
+    tableOnPage.classList.toggle('hidden');
+    let viewGraphButton =document.querySelector('#viewGraph');
+    viewGraphButton.classList.toggle('hidden');
+    let viewTableButton = document.querySelector('#viewTable');
+    viewTableButton.classList.toggle('hidden');
+}
+
+var displayGraphButton = document.querySelector('#viewGraph');
+displayGraphButton.addEventListener ('click', function(e) {
+    e.preventDefault();
+    toggleGraphDisplay();
+});
+
+var displayTableButton = document.querySelector('#viewTable');
+displayTableButton.addEventListener ('click', function(e) {
+    e.preventDefault();
+    toggleGraphDisplay();
+});
```

Comment:
This isn't very dry. Since both listeners do the same thing could you modify the toggleGraphDisplay function so it can be directly used as a callback for your event listeners?

Reply:
Yes, done it now